README.rst

```diff
@@ -19,6 +19,12 @@ Schwifty is a Python library for working with BICs and IBANs. It allows you to
 * access all relevant components as attributes
 
 
+Versioning
+----------
+
+Since the mapping from BIC to bank_code is updated from time to time, Schwifty uses
+`CalVer <http://www.calver.org/>`_ with the scheme ``YY.0M.Micro``.
```

Comment:
`0M`? Shouldn't this be `MM`?

Reply:
`MM` mean no leading zero - like `twisted`. For this month it would be `2018.3` whereas `0M` would have a leading zero like `Ubuntu`, the example would be `2018.03`. Both are valid and I like the latter more.

But if a majority votes for `MM`, we can of course switch. Let's for people who have strong opinions on this topic *hint*, *hint*